docs/dashboard-layouts.md

```diff
@@ -30,7 +30,7 @@ On first template requests, if a user does not have any custom template, new is
 
 The newly generated template becomes the user default.
 
-Each template will contain 4 variants of the configurations bad on screen sizes.
+Each template will contain 4 variants of the configurations based on various screen sizes.
```

Comment:
@Hyperkid123 I'm noticing locally it is impossible to `copy` the base-template as the copy API expects an int ID. 

For example: `POST http://localhost:8000/api/chrome-service/v1/dashboard-templates/base-template/copy` results in 400 bad request as `base-template` cannot be converted into an int.

Reply:
Yeah, you can't copy the base as it will be forked for a user when they request a specific dashboard template and it does not exist yet.

That said, this feature will be required once we allow multiple versions of the same template for a user.